src/fairchem/core/common/relaxation/ase_utils.py

```diff
@@ -104,8 +109,9 @@ def __init__(
 
         if model_name is not None:
             if local_cache is None:
-                logging.error("Local cahce must be set when using model name")
-                return
+                raise NotImplementedError(
+                    "Local cache must be set when specifying a model name"
```

Comment:
It might be useful to warn if both model_name and checkpoint path are given. It looks like right now the model_name will take precedence and silently ignore checkpoint_path if given